command/title_body_survey.go

```diff
@@ -146,70 +146,57 @@ func selectTemplate(nonLegacyTemplatePaths []string, legacyTemplatePath *string,
 	return string(templateContents), nil
 }
 
-func titleBodySurvey(cmd *cobra.Command, issueState *issueMetadataState, apiClient *api.Client, repo ghrepo.Interface, providedTitle, providedBody string, defs defaults, nonLegacyTemplatePaths []string, legacyTemplatePath *string, allowReviewers, allowMetadata bool) error {
-	editorCommand, err := determineEditor(cmd)
-	if err != nil {
-		return err
+func titleBodySurvey(cmd *cobra.Command, issueState *issueMetadataState, apiClient *api.Client, repo ghrepo.Interface, askTitle, askBody bool, defs defaults, nonLegacyTemplatePaths []string, legacyTemplatePath *string, allowReviewers, allowMetadata bool) error {
```

Comment:
👏 I think the booleans (askTitle and askBody) in this function improve readability and help trace what the body of the PR will be. Nice work @AliabbasMerchant

Reply:
Thanks :smile: